README.md

```diff
@@ -73,6 +73,7 @@ tests/              - test cases package
 | ------ | ----------- | -------- | -------------
 | GET    | `/products` | List     | Returns all the products in the databse (can be filtered by a query string)
 | POST   | `/products` | Create   | Create a new product, and upon success, receive a Location header specifying the new order's URI
+| PUT   | `/products` | Update   | Update fields of a existing product
```

Comment:
the url should be /products/<product_id>

Reply:
but this put method will have a payload anyways

Reply:
yes, but the api need product_id to locate witch Product will be updated.